projects/ADuCM360_demo_cn0411/.gitignore

```diff
@@ -0,0 +1,2 @@
+/Debug/
+/Release/
```

Comment:
There already is a .gitignore on the EVAL-ADICUP360 repository. No need to add a new one inside the project.

Reply:
You're right. I took as a template other projects that contain the file (all of them). I'm not sure which is the purpose of having it in each folder.